blockstack/lib/storage/crawl.py

```diff
@@ -196,14 +197,14 @@ def remove_atlas_zonefile_data( zonefile_hash, zonefile_dir ):
     return True
 
 
-def add_atlas_zonefile_data( zonefile_text, zonefile_dir ):
+def add_atlas_zonefile_data(zonefile_text, zonefile_dir, fsync=True):
     """
     Add a zone file to the atlas zonefiles
     Return True on success
     Return False on error
     """
 
-    rc = store_atlas_zonefile_data( zonefile_text, zonefile_dir )
+    rc = store_atlas_zonefile_data(zonefile_text, zonefile_dir, fsync=True)
```

Comment:
not a bug, because no callers of `add_atlas_zonefile_data` use the optional arg, but I assume the intent was to pass the `fsync` option through?

Reply:
Yes; will fix.